static/app/utils/withIssueTags.tsx

```diff
@@ -34,7 +34,10 @@ const getUsername = ({isManaged, username, email}: User) => {
 function withIssueTags<P extends InjectedTagsProps>(
   WrappedComponent: React.ComponentType<P>
 ) {
-  class WithIssueTags extends React.Component<P, State> {
+  class WithIssueTags extends React.Component<
+    Omit<P, keyof InjectedTagsProps> & Partial<InjectedTagsProps>,
+    State
+  > {
```

Comment:
Updating `WithIssueTags` type to fix typescript complaining about missing `tags` prop

Reply:
Hmm I don't think I understand why ts is complaining?

Reply:
The props for components returned by `withIssueTags` should be equal to the props of the child component minus the `tags` prop since `withIssueTags` supplies `tags` to the child.

Updated the type construction on `withIssueTags` to omit `tags` since it wasn't doing this before and invoking a component  with `withIssueTags` would cause typescript to complain about missing `tags`.